source/jquery.flot.navigate.js

```diff
@@ -117,6 +117,9 @@ can set the default in the options.
             frameRate: 60,
             mode: 'smart'
         },
+        recenter: {
+            interactive: false
```

Comment:
This just occurred to me, but as is this is a breaking change, as all scripts that have set pan to be interactive will no longer be able to recenter on double-click. I think perhaps we should move this option into the pan options, and default it to true. The new pan option could be called 'recenterOnDoubleClick' (or maybe just 'allowRecenter'), and would only work when pan.interactive was also set to true. I think this makes some sense as re-centering is a pan operation of sorts. The key here is to prevent a breaking change, which doesn't seem necessary for the feature.

Reply:
Agree and fixed.